docker/service/start_service.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# start_service.sh
```

Comment:
Unfortunately we aren't consistent with using `_` or `-`...I'd prefer `-`, but consistency is more important...